tests/integration/targets/consul/tasks/main.yml

```diff
@@ -10,8 +10,8 @@
 
 - name: Install Consul and test
   vars:
-    consul_version: 1.5.0
-    consul_uri: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/consul/consul_{{ consul_version }}_{{ ansible_system | lower }}_{{ consul_arch }}.zip
+    consul_version: 1.13.2
+    consul_uri: https://releases.hashicorp.com/consul/{{ consul_version }}/consul_{{ consul_version }}_{{ ansible_system | lower }}_{{ consul_arch }}.zip
```

Comment:
@mattclay is it possible to update https://s3.amazonaws.com/ansible-ci-files/ with a newer version? Or should we stop using it and use the vendor's server instead?

Reply:
I recommend using the vendor's server. We switched a lot of tests to use our S3 bucket before the collection split, mostly due to unreliable servers. Hopefully that's no longer an issue and the official sources can be used instead.

Reply:
Sounds good! Thanks.